### ISSUE TITLE
doctest of postprocessing will break when the module name changes

### ISSUE THREAD

**Issue by nikohansen** [numbbo/coco#373]
Maybe there is a trick to make the doctest independent of the module name?

**Comment by nikohansen** [comment]
Given we might have in not so far future a new `cocoex2` module, we should consider a design which goes smoothly with this development. Should we name the current to `cocoex0`?

**Comment by brockho** [comment]
Not sure what you have in mind what `cocoex2` will be: I guess you mean `cocopp2`? And even then it is not clear whether this refers to the planned rewrite of the postprocessing. What comes first into my mind is to suggest using `cocopp2` for the first renaming from `bbob_pproc` because the change will be effective in version 2.0 of COCO and then have finally the rewrite named as `cocopp`.

**Comment by nikohansen** [comment]
> I guess you mean cocopp2?

yes, sorry. `cocopp2` is also suggestive for the dependency on Python 2. `cocopp` as the successor of `cocopp2` seems a little counterintuitive, so maybe `cocopp3`?
